### ISSUE TITLE
Unable to load into world due to vault crashing my game

### ISSUE THREAD

**Issue by Jawz145** [Iskallia/Vault-public#472]
I attempted to load into a raffle vault, my game crashed upon exiting the portal, the chunks began to load prior to this and now I can't load back into my world

Steps to reproduce the behaviour:
1. Insert Raffle Crystal into portal frame
2. Enter Portal
3. Wait for the vault to load and enter when it did
4. Crash, try to load world and get stuck on loading at 100% with maxed out RAM

I expected to load into the raffle vault and complete it

I have attached my last crash report -> 
[crash-2021-10-16_21.42.00-server.txt](https://github.com/Iskallia/Vault-public/files/7358332/crash-2021-10-16_21.42.00-server.txt)

- World Type: Singleplayer that was opened to LAN
- The_Vault Version is 1.5

**Additional context**
I worked with a person within the discord to try and restore the world, deleted the vault dimension files to see if this would send me back and made it so I kept inventory as I didn't know if I'd die unfairly but the files were just recreated, I edited the playerdata via nbtexplorer to set my dimension to overworld and the coords to 0,70,0 and no luck so restored the original playerdata. I have restored the vault dimension files I removed to so it's back to what it was

**Comment by FortThePie** [comment]
So i found a solution, kind of, if you delete the world data file it will load, assuming you are on a world in single player you can just cheat your level back up and all your other stuff should still exist

**Comment by Jawz145** [comment]
@FortThePie which is the world data file? XD I don't wanna delete the wrong one lol

**Comment by FortThePie** [comment]
The file that just says data is the one, i have found some issues the come with doing this, the vaults wont load, which will fix if you just go into a vault do the command that kicks you from the vault, then repeat until you find a vault that loads

**Comment by Jawz145** [comment]
Does it not load forever or does it start working permanently after you successfully load into one? And what's that command please?

**Comment by FortThePie** [comment]
Once one works it will keep working, the command is /the_vault debug and than just do the auto fill command for after debug

**Comment by Jawz145** [comment]
okay thanks, i'll try it and let you know if it is successful

**Comment by Jawz145** [comment]
Still stuck at 100% unfortunately, it just gets to that point and hangs forever

**Comment by FortThePie** [comment]
Weird cause it worked for me and i had the same issue

**Comment by Jawz145** [comment]
OH, no it just changed to joining world! there's more progress than before so lets see if it loads further

**Comment by FortThePie** [comment]
I also have found that it will break again eventually, there is definitely an issue with the mod that causes it to break

**Comment by FortThePie** [comment]
I am going to try to find what file in data needs to be deleted to fix the issue so when it breaks you dont need to delete the entire folder

**Comment by Jawz145** [comment]
I assumed as much, regardless it crashed again unfortunately, I have attached the log file again
[crash-2021-10-17_14.56.29-server.txt](https://github.com/Iskallia/Vault-public/files/7360025/crash-2021-10-17_14.56.29-server.txt)

**Comment by Jawz145** [comment]
Oop my bad, was because i didnt change the ram, let me try again

**Comment by FortThePie** [comment]
At least it got past 100% thats progress

**Comment by FortThePie** [comment]
Oh lol

**Comment by Jawz145** [comment]
> At least it got past 100% thats progress

exactly, a crash log is way easier to read then just a loading screen XD

**Comment by Jawz145** [comment]
WE'RE BACK IN BUSINESS BABY!!!! God it took a lot of trial and error about putting the files back in and out of the different folders, and I mean ALL folders but I'm back and we're running vaults again XD thanks!

**Comment by Jawz145** [comment]
And even better still. the vault just worked first time!! noice one

**Comment by Tomzor** [comment]
Hey dredging this up cause I ran into the same issue,

Deleting all the files in the data file of the save of the offending world fixes the problem however what I found strange is that when I put a backup of the data file back in after loading the world it continued on fine so I have even fewer ideas as to what has broke here.
